packages/datagateway-common/src/table/columnFilters/textColumnFilter.component.test.tsx

```diff
@@ -12,7 +12,7 @@ import userEvent from '@testing-library/user-event';
 
 jest.mock('../../api');
 jest.useFakeTimers('modern');
-const DEBOUNCE_DELAY = 250;
+const DEBOUNCE_DELAY = 500;
```

Comment:
It might be worth moving this variable to the actual component, and export it to use it in both the tests and when passing the actual debounce - it means in the future if we edit the debounce we only have to change it in one place :) Just an opportunity to refactor the code to be a little more future-proof

Otherwise good work!

Reply:
Thanks Louise! Okay I understand, I've made and pushed the change.